reconcile/aws_iam_password_reset.py

```diff
@@ -8,9 +10,23 @@
 QONTRACT_INTEGRATION = "aws-iam-password-reset"
 
 
+def get_roles(roles: list[dict[str, Any]], org_username: str) -> dict[str, Any]:
+    return [d for d in roles if d["org_username"] == org_username][0]
+
+
+def account_in_roles(roles: list[dict[str, Any]], aws_account: str) -> bool:
+    for role in roles:
+        if role.get("aws_groups"):
+            for g in role.get("aws_groups"):
```

Comment:
Shorter:
```suggestion
        for g in role.get("aws_groups") or []:
```
(and re-indent)